test/projectManager.ts

```diff
@@ -113,28 +135,79 @@ export class ProjectManager {
     /**
      * Adds a plugin to a Cordova project.
      */
-    public static addPlugin(projectFolder: string, plugin: string): Q.Promise<void> {
+    public static addPlugin(projectFolder: string, plugin: string): Q.Promise<string> {
         return ProjectManager.execAndLogChildProcess("cordova plugin add " + plugin, { cwd: projectFolder });
     }    
 
     /**
      * Builds a specific platform of a Cordova project. 
      */
-    public static buildPlatform(projectFolder: string, targetPlatform: platform.IPlatform): Q.Promise<void> {
-        return ProjectManager.execAndLogChildProcess("cordova build " + targetPlatform.getCordovaName(), { cwd: projectFolder });
+    public static buildPlatform(projectFolder: string): Q.Promise<string> {
+        var targetPlatform = platform.PlatformResolver.resolvePlatform(tu.TestUtil.readTargetPlatform());
+        
+        // don't print output here because the iOS build process outputs so much nonsense that it buffer overflows and exits the entire test process
```

Comment:
An alternative here would have been increasing the [buffer size](https://github.com/Microsoft/cordova-plugin-code-push/pull/96/files#diff-8a098a7d94b3a7fe31827cde0cfeb7b6L190). However, I agree the build output is not useful during tests.